spiffworkflow-backend/src/spiffworkflow_backend/services/process_instance_service.py

```diff
@@ -443,6 +443,8 @@ def save_file_data_and_replace_with_digest_references(
         models = cls.replace_file_data_with_digest_references(data, process_instance_id)
 
         for model in models:
+            if current_app.config["SPIFFWORKFLOW_BACKEND_PROCESS_INSTANCE_FILE_DATA_FILESYSTEM_PATH"] is not None:
+                model.store_file_on_file_system()
```

Comment:
I see, so you want to retain the old method and keep it as default?

I guess it's a good way to test things out, but ultimately I would advise against keeping file storage in the database as a sane default, it's not sane =].